src/client/Client.h

```diff
@@ -446,6 +473,9 @@ class Client : public Dispatcher, public md_config_obs_t {
 	    int mask=CEPH_STAT_CAP_INODE_ALL);
   int fstatx(int fd, struct ceph_statx *stx, const UserPerm& perms,
 	     unsigned int want, unsigned int flags);
+  int fstatxat(int dirfd, const char *relpath,
```

Comment:
```suggestion
  int statxat(int dirfd, const char *relpath,
```
?

Also: shouldn't `Client::statx` just call return `statxat(CEPHFS_AT_FDCWD, ...);` as we do for `::open`?

Reply:
Ditto for other `*at` calls.